client/src/templates/Challenges/components/help-modal.tsx

```diff
@@ -53,61 +89,210 @@ function HelpModal({
+              closeHelpModal();
+            }}
+            ref={formRef}
+          >
+            <div
+              style={{
+                display: 'flex',
+                flexDirection: 'column'
+              }}
+            >
+              <fieldset>
+                <legend style={{ fontSize: 'large' }}>
+                  {t('learn.must-confirm-statements')}
+                </legend>
+                <CheckboxHelpModal
+                  name='read-search-ask-checkbox'
+                  i18nkey={t('learn.read-search-ask-checkbox')}
+                  onChange={event =>
+                    handleCheckboxChange(event, setReadSearchCheckbox)
+                  }
+                  value={readSearchCheckbox}
+                  href={RSA}
+                  t={t}
+                />
+                <Spacer size='small' />
+                <CheckboxHelpModal
+                  name='similar-questions-checkbox'
+                  i18nkey={t('learn.similar-questions-checkbox')}
+                  onChange={event =>
+                    handleCheckboxChange(event, setSimilarQuestionsCheckbox)
+                  }
+                  value={similarQuestionsCheckbox}
+                  href={generateSearchLink(challengeTitle, challengeBlock)}
+                  t={t}
+                />
+              </fieldset>
+              <label htmlFor='help-modal-form-description'>
+                {t('forum-help.whats-happening')}
+                <span className='sr-only'>{t('learn.min-50-max-500')}</span>
+              </label>
+              <FormControl
+                id='help-modal-form-description'
+                name='description'
+                onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
+                  setDescription(event.target.value);
+                }}
+                componentClass='textarea'
+                required={true}
+                style={{
+                  height: 100
+                }}
+                value={description}
+                maxLength={DESCRIPTION_MAX}
+                type='text'
+              />
+              {description.length < DESCRIPTION_MIN ? (
+                <p className='chars-left'>
+                  {t('learn.minimum-characters').replace(
+                    '{{characters}}',
+                    (DESCRIPTION_MIN - description.length).toString()
+                  )}
+                </p>
+              ) : (
+                <p className='chars-left'>
+                  {t('learn.characters-left').replace(
+                    '{{characters}}',
+                    (DESCRIPTION_MAX - description.length).toString()
+                  )}
+                </p>
+              )}
+            </div>
+            <Spacer size='xxSmall' />
+
+            <Button
+              block={true}
+              size='large'
+              variant='primary'
+              type='submit'
+              disabled={!canSubmitForm()}
+            >
```

Comment:
```suggestion
            <Button
              block={true}
              size='large'
              variant='primary'
              type='submit'
              disabled={!canSubmitForm()}
              {...(!canSubmitForm() && { tabIndex: '-1' })}
            >
```

If this button is truly disabled then it cannot accept keyboard focus.